export default [
	{
		name: 'Mark Gerstein, PhD',
		affiliation: 'Yale University',
		url: 'http://resource.psychencode.org/',
		image: '/assets/MCBIOS19_files/MarkGerstein200x300.jpg',
		heading: 'Brain Genomics',
		body: `<div class="elementor-widget-container">
        <div class="elementor-text-editor elementor-clearfix">Despite progress in defining genetic risk for psychiatric disorders, their molecular mechanisms remain elusive. Addressing this, the PsychENCODE Consortium has generated a comprehensive online resource for the adult brain across 1866 individuals. The PsychENCODE resource contains ~79,000 brain-active enhancers, sets of Hi-C linkages, and topologically associating domains; single-cell expression profiles for many cell types; expression quantitative-trait loci (QTLs); and further QTLs associated with chromatin, splicing, and cell-type proportions. Integration shows that varying cell-type proportions largely account for the cross-population variation in expression (with &gt;88% reconstruction accuracy). It also allows the building of a gene regulatory network, linking genome-wide association study variants to genes (e.g., 321 for schizophrenia). We embed this network into an interpretable deep-learning model, which improves disease prediction by ~6-fold versus polygenic risk scores and identifies key genes and pathways in psychiatric disorders. <br/><br/> After graduating from Harvard with a A.B. in physics in 1989, Prof. Mark Gerstein earned a doctorate in theoretical chemistry and biophysics from Cambridge University in 1993. He did postdoctoral research in bioinformatics at Stanford University from 1993 to 1996. He came to Yale in 1997 as an assistant professor in the Department of Molecular Biophysics and Biochemistry, and since 1999, in the Computer Science Department. He was named an associate professor in 2001, and the following year became co-director of the Yale Computational Biology and Bioinformatics Program. Gerstein has published appreciably in the scientific literature, with &gt;400 publications in total, including a number of them in prominent venues, such as Science, Nature, and Scientific American. His research is focused on bioinformatics, and he is particularly interested in data science &amp; data mining, macromolecular geometry &amp; simulation, human genome annotation &amp; disease genomics, and genomic privacy.&nbsp;</div>
    </div>`
	},
	{
		name: 'King Jordan',
		affiliation: 'Georgia Institute of Technology',
		image: '/assets/MCBIOS19_files/jordan_king.jpg',
		heading: 'Genomic medicine for understudied populations: lessons from Colombia',
		body: `<div class="elementor-widget-container">
        <div class="elementor-text-editor elementor-clearfix"><p>Genomic medicine promises to revolutionize healthcare. However, genomics research remains heavily biased towards European-ancestry cohorts, while understudied populations bear a disproportionate disease burden. This genomics research gap has the potential to exacerbate existing health disparities. Closing the gap will require: (i) more genomics research on diverse populations, (ii) better methods so that insights from current studies can be applied across populations, and (iii) development of local capacity so that genomic approaches to healthcare can be implemented worldwide. Our group is working on these three areas together with collaborators from the country of Colombia in South America. Our research and development efforts are focused on neighboring populations in Colombia with distinct ancestry profiles: Antioquia (European) and Chocó (African). We are investigating the role that genetic ancestry plays in the epidemiology of type 2 diabetes (T2D) and patient drug response (pharmacogenomics) in these two populations. Chocó has significantly higher predicted genetic risk for T2D compared to Antioquia, and the elevated genetic risk for T2D in Chocó is correlated with higher African ancestry. Despite its higher genetic risk, the population of Chocó has a three-times lower observed T2D prevalence than Antioquia, indicating that environmental factors better explain differences in T2D outcomes for Colombia. Our exploration of dietary and lifestyle factors in Chocó illustrates how low socioeconomic status can divergently manifest as a T2D protective factor in developing countries. A number of pharmacologically relevant genetic variants show anomalous allele frequencies within and between the two Colombian populations, and these differences are also associated with their distinct genetic ancestry profiles. For example, one particular variant in the SLCO1B1 gene (rs4149056), which is associated with an increased risk of toxicity to a commonly prescribed statin, is found at relatively high frequency in Antioquia and associated with European ancestry. We have developed and validated an inexpensive allele-specific PCR assay to test for the presence of such population-enriched pharmacogenomic variants in resource-limited settings like Colombia. We hope that our population-centered approach can serve as a model for the application of genomic methods to healthcare in understudied populations worldwide.</p></div>
    </div>`
	},
	{
		name: 'Predrag Radivojac',
		affiliation: 'Northeastern University',
		image: '/assets/MCBIOS19_files/PredragRadivojac200x300.jpg',
		heading:
			'Semi-Supervised Learning to the Rescue: Towards Deep Understanding of Coding Variant Effects',
		body: `<div class="elementor-element elementor-element-1d69c7d elementor-widget elementor-widget-text-editor" data-id="1d69c7d" data-element_type="widget" data-widget_type="text-editor.default">
        <div class="elementor-widget-container">
            <div class="elementor-text-editor elementor-clearfix"><p>A major goal in computational biology is the development of algorithms, analysis techniques, and tools towards deep mechanistic understanding of life at a molecular level. In the process, computational biology must take advantage of the new developments in artificial intelligence and machine learning, and then extend beyond pattern analysis to provide testable hypotheses for experimental science. This talk will focus on our recent contributions to understanding molecular mechanisms of disease and relevant related work. We will first discuss the development of semi-supervised machine learning techniques for partially observable domains such as molecular biology; that is, methods for accurate estimation of frequency of occurrence of hard-to-measure and rare events. We will then show how these methods play key roles in inferring protein cellular roles and phenotypic effects of genomic mutations, with an emphasis on understanding the molecular mechanisms of human genetic disease. We assessed the value of these methods in the wet lab where we tested the molecular mechanisms behind selected de novo mutations in a cohort of individuals with neurodevelopmental disorders. We will conclude by discussing implications of this work on future research in machine learning, genome interpretation, and precision health.</p></div>
        </div>
        </div>
        
        <div class="elementor-element elementor-element-e225d77 elementor-widget elementor-widget-text-editor" data-id="e225d77" data-element_type="widget" data-widget_type="text-editor.default">
				<div class="elementor-widget-container">
					<div class="elementor-text-editor elementor-clearfix"><p><strong>Predrag Radivojac</strong> is a Professor of Computer Science at Northeastern University, where he recently moved from Indiana University. Prof. Radivojac received his Bachelor’s and Master’s degrees in Electrical Engineering from the University of Novi Sad and University of Belgrade, Serbia. His Ph.D. degree is in Computer Science from Temple University (2003) under the direction of Prof. Zoran Obradovic and co-direction of Prof. Keith Dunker. In 2004 he held a post-doctoral position in Keith Dunker’s lab at Indiana University School of Medicine, after which he joined Indiana University Bloomington. Prof. Radivojac’s research is in the areas of computational biology and machine learning with specific interests in protein function, MS/MS proteomics, genome interpretation, and precision health. He received the National Science Foundation (NSF) CAREER Award in 2007 and is an August-Wilhelm Scheer Visiting Professor at Technical University of Munich (TUM) as well as an honorary member of the Institute for Advanced Study at TUM. At Indiana University, he was Associate Chair of the Department of Computer Science and a co-Director of all of Informatics and Data Science for the multi-campus Precision Health Initiative. Prof. Radivojac’s projects have been regularly supported by NSF and National Institutes of Health (NIH). He is currently an Editorial Board member for the journal Bioinformatics, Associate Editor for PLoS Computational Biology, and serves his third term (elected) on the Board of Directors of the International Society for Computational Biology (ISCB).</p></div>
				</div>
				</div>`
	},
	{
		name: 'Josh F. Peterson',
		affiliation: 'Vanderbilt University Medical Center',
		image: '/assets/MCBIOS19_files/Peterson_Josh_2_3.jpg',
		heading: 'Clinical Applications of Genomic Medicine: The State of the Art',
		body: `<div class="elementor-element elementor-element-e453e29 elementor-widget elementor-widget-text-editor" data-id="e453e29" data-element_type="widget" data-widget_type="text-editor.default">
        <div class="elementor-widget-container">
            <div class="elementor-text-editor elementor-clearfix"><p>&nbsp;The field of genomic medicine has been transformed by dramatic reductions in the cost and ease of obtaining clinical-grade genotyping or sequencing data. &nbsp;Interest in implementing pharmacogenomics to tailor medical therapies, sequencing to screen for Mendelian traits, and application of polygenic risk scores to predict incident chronic diseases has surged.&nbsp;&nbsp; What has lagged the advances in the laboratory is the technical ability to broadly and repeatedly access stored genomic data for patient care, and to deliver evidence based and patient context sensitive recommendations for treatment or further screening once genetic risks are identified. &nbsp;This talk will review the progress that is being made in translating newer applications in germline genomic medicine to routine patient care.&nbsp; We will discuss the landscape of NIH-funded projects pioneering large scale genetic population screening and implementation, and review the remaining challenges to making these technologies accessible to all.</p></div>
        </div>
        </div>
        
        <div class="elementor-element elementor-element-2998ec6 elementor-widget elementor-widget-text-editor" data-id="2998ec6" data-element_type="widget" data-widget_type="text-editor.default">
				<div class="elementor-widget-container">
					<div class="elementor-text-editor elementor-clearfix"><p><strong>Josh Peterson,</strong> MD, MPH is Associate Professor with appointments in the Departments of Biomedical Informatics and Medicine at Vanderbilt University Medical Center (VUMC), and an internist with an active primary care practice. Dr. Peterson’s research interests are in precision medicine with a focus on translating genomic technologies to routine clinical care. Over his 20-year career in clinical informatics, he has led the design and implementation of clinical decision support systems to improve drug safety. Currently, he leads implementation of one of the largest pharmacogenomics implementations in the US – PREDICT. He serves as a principle investigator for an NIH funded project to simulate the clinical impact and cost-effectiveness of sequencing large populations over their lifetime. He also leads two NIH-sponsored genomic medicine consortia: eMERGE (Electronic Medical Records and Genomics) where he is principle investigator of the Coordinating Center and co-Chair of the Outcomes Workgroup and IGNITE 2 where he serves as principle investigator for the VUMC site.</p></div>
				</div>
				</div>`
	}
];
